jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/surfaces/AccordionTests.java

```diff
@@ -3,33 +3,28 @@
 import io.github.epam.TestsInit;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.accordionFrame;
-import static io.github.com.StaticSite.surfaceAccordionDisabledPage;
-import static io.github.com.StaticSite.surfaceAccordionPage;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.surfaces.AccordionPage.*;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
 
```

Comment:
Please add comment about link to material ui site element like in https://github.com/jdi-testing/jdi-light/pull/2875/files